packages/wallets/src/components/Base/WalletAlertMessage/WalletAlertMessage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactNode } from 'react';
+import useDevice from '../../../hooks/useDevice';
+import ErrorCircleCrossmark from '../../../public/images/error-circle-crossmark.svg';
+import InfoCircleDots from '../../../public/images/info-circle-dots.svg';
+import SuccessCircleCheckmark from '../../../public/images/success-circle-checkmark.svg';
+import { WalletText } from '../index';
+import styles from './WalletAlertMessage.module.css';
+
+const typeMapper = {
```

Comment:
minor/optional: I tend to prefer simple types over extra structures (as every extra structure generates additional overhead when reading code)